package.json

```diff
@@ -146,6 +146,8 @@
     "lodash.debounce": "4.0.8",
     "postscribe": "^2.0.8",
     "preact": "^10.4.4",
+    "preact-render-spy": "^1.3.0",
+    "preact-render-to-json": "^3.6.6",
```

Comment:
cc @nickytonline

Reply:
@sarthology and @narender2031, we no longer use `preact-render-spy` and p`react-render-to-json`. Merge latest of master to ensure everything is up to date.

There is a [changelog post on DEV](https://dev.to/devteam/changelog-frontend-edition-30l7) detailing all the changes.